General-Application-Information.md

```diff
@@ -122,6 +123,39 @@ The Git organization usually has very limited mentoring capacity.
 These days we usually accept between 0 and 3 students per season
 (Winter or Summer).
 
+## Note about giving back and mentoring
+
+We appreciate very much students and interns who stay around after the
+mentoring period is over. It is very nice to see them on the mailing
+list, even if they don't contribute much. It's of course better when
+they continue to contribute though, even by just reviewing a patch
+from time to time.
+
+Some people have been around for more than 10 years, others have
+become regular contributors and that's great!
+
+One very nice way to contribute and to give back is to mentor or
+co-mentor other students or interns coming after you. It helps create
+more opportunities for more students and interns like you, as
+mentoring capacity is the main factor preventing us from accepting
+more students and interns. If each student or intern accepted to
+co-mentor twice (once in Summer and once in Winter) just after they
+have been mentored, our mentoring capacity could increase
+significantly each year.
+
+Unfortunately very few former students or interns have been willing to
+just co-mentor along with an experienced mentor, even though it
+doesn't require much work, technical ability or time.
```

Comment:
I don't feel like this is quite right.  Being a good mentor does require work and time, and I think a mentor should be able to invest a few hours a week for mentoring.  Depending on the student that may not always be necessary, but every mentor should be able to dedicate that time.  For some people that might not seem a lot of work, but for others it is.

I think it would be better to write down our expectations for mentorship, rather than a blanket "doesn't require much work".

Similarly, I think a mentor should have some technical ability, even though probably less is required from a co-mentor.  But they should be able to navigate the git codebase, and help a student find their way through.  I think a lot of students are able to do that after completing GSoC or Outreachy, but I'd rather phrase it as "Successful former students/interns usually have adequate technical ability to be a successful mentor".

Reply:
Yeah, you are right. I think it will be better with what you suggest.